### DISCUSSION TITLE
Specify arbitrary commands that only run on (re)build

### DISCUSSION THREAD

**Discussion by chrisjsewell** [tox-dev/tox#2958] [Q&A] [answered]
Heya,
You have `commands`/`pre_commands`/`post_commands` that run everytime you call tox,
and `install_command` that only runs on the environment (re)build.
But is there a way to get one or more other arbitrary commands to run only on environment (re)build (after install)?

**Comment by gaborbernat** [comment, marked as answer]
Not part of the core but you create a plugin for it. Note install command is mostly deprecated and shouldn't be used for customization.

**Reply by chrisjsewell** [reply]
Yeh cheers, I see this is kinda mentioned in https://github.com/tox-dev/tox/issues/715#issuecomment-1012018435
The problem is I heavily use tox-conda, which doesn't look like it will be updated to tox4 anytime soon 😢

**Reply by chrisjsewell** [reply]
Bit of a rock and a hard place, but thanks for the quick reply anyway
